public/app/features/explore/QueryField.tsx

```diff
@@ -313,32 +313,35 @@ export class QueryField extends React.PureComponent<QueryFieldProps, QueryFieldS
 
   handleEnterAndTabKey = (event: KeyboardEvent, change: Change) => {
     const { typeaheadIndex, suggestions } = this.state;
-    if (this.menuEl) {
-      // Dont blur input
-      event.preventDefault();
-      if (!suggestions || suggestions.length === 0) {
-        return undefined;
-      }
+    if (!event.shiftKey) {
```

Comment:
personally I usually try to limit code nesting, as it usually leads to more readable code. So in this case I would just check if (event.shiftKey) and return (at the top of function), instead of indenting everything in this negative if case.

Reply:
Yeah, I'd exit early here too.

Reply:
definitely agree, i usually try to limit nesting as well but seem to have missed it this time, oops! will push another commit.